templates/taxi/car_form.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>Create car</h1>
```

Comment:
I think will be better to use `objects | yesno` statement cause you have always "Create car" in both variants: create / update.